framework/aster-frame/Cargo.toml

```diff
@@ -15,6 +15,7 @@ buddy_system_allocator = "0.9.0"
 cfg-if = "1.0"
 gimli = { version = "0.28", default-features = false, features = ["read-core"] }
 inherit-methods-macro = { git = "https://github.com/asterinas/inherit-methods-macro", rev = "98f7e3e" }
+xarray = {git = "https://github.com/asterinas/XArray"}
```

Comment:
I stand against developing xarray in a separate repository, for the correctness of vmo heavily depends on the invariants provided by xarray. By making it in a separate repository, it is easy for following developers of Asterinas neglect viewing the code of xarray.

And, CI, license and contribution management of the xarray repository would be separatedly maintained. We shouldn't doing so repeatedly when we have robust CI infrastructures in the main repo. We should soon setup a publishing workflow to easily publish Asterinas crates, which is the sole benefit of developing xarray in the standalone repository.

Reply:
I agree that the implementation of Vmo is closely related to the specific properties of XArray. The reason for developing XArray in a separate repo because its functionality is completely independent. Being a standalone crate makes it more accessible for external developers to use and allows for testing in a standard environment. BTW, it would indeed be more convenient if there were a more efficient process for releasing crates related to Asterinas in the future~

Reply:
Indeed, being a standalone crate does not conflict with being maintained in the same repository. Generally, I agree with you that XArray can be a standalone crate, but I agree with @junyang-zh that maintaining this crate separately might not be a good idea.